en/development/testing.rst

```diff
@@ -12,13 +12,25 @@ Installing PHPUnit
 
 CakePHP uses PHPUnit as its underlying test framework. PHPUnit is the de-facto
 standard for unit testing in PHP. It offers a deep and powerful set of features
-for making sure your code does what you think it does. PHPUnit can be installed
-through the `pear installer <http://pear.php.net>`_. To install PHPUnit run the
-following::
+for making sure your code does what you think it does.::
+
+As of Dec. 31st, 2014 installation via pear.phpunit.de is no longer valid.
```

Comment:
I don't think this is necessary, as time travellers from the past are pretty infrequent :smile: